index.html

```diff
@@ -756,8 +758,7 @@ <h2>Extensions to the `Navigator` interface</h2>
     </section>
     <section>
         <h1>Declaring candidate regions for default direct manipulation behaviors</h1>
```

Comment:
Perhaps remove the "default" in this section header too?

Reply:
good idea, done